stack.yaml

```diff
@@ -1,10 +1,10 @@
 flags: {}
 packages:
 - '.'
-- 'yesod-markdown'
 # - location:
 #     git: git@github.com:robgssp/yesod-markdown
 #     commit: 1860815cf5fe193e9593a9adc7211929e803a7d1
 extra-deps:
 - ldap-client-0.1.0
+- yesod-markdown-0.10.0
```

Comment:
Should this be in the extra-deps section, or just be like the other dependencies?